apps/src/code-studio/initApp/project.js

```diff
@@ -514,16 +513,12 @@ var projects = (module.exports = {
     );
   },
 
-  // Currently, only applab when the experiment is enabled. Hide if
-  // hideShareAndRemix is set on the level.
+  // Currently, only applab and gamelab.
```

Comment:
What is the roll-out plan for this feature?

Reply:
@simplycloud  is on point for deciding the specific timing. Merging this PR will start the external Beta for Android export.